OracleFMWInfrastructure/README.md

```diff
@@ -72,7 +74,7 @@ If you need to find the passwords at a later time, grep for "password" in the Do
 ### Write your own Oracle Fusion Middleware Infrastructure domain with WLST
 The best way to create your own, or extend domains is by using [WebLogic Scripting Tool](https://docs.oracle.com/middleware/1221/cross/wlsttasks.htm). You can find an example of a WLST script to create domains at [createInfraDomain.py](dockerfiles/12.2.1.2/container-scripts/createInfraDomain.py). You may want to tune this script with your own setup to create DataSources and Connection pools, Security Realms, deploy artifacts, and so on. You can also extend images and override an existing domain, or create a new one with WLST.
 
-## Building and Running the Oracle FMW Infrastructure Docker Image
+## Running the Oracle FMW Infrastructure Domain Docker Image
 To try a sample of a FMW Infrastructure Domain image configured, you will need two images, the FMW Infrastructure Domain image and an Oracle Database image. The Oracle Database image can be pulled from the [DockerStore](https://store.docker.com/images/oracle-database-enterprise-edition) or the [Oracle Container Registry](https://container-registry.oracle.com) or you can build your own using the Dockerfiles and scripts in GitHub. 
```

Comment:
DB needs to be running somewhere (regardless whether in another container or on some other remote server), and credentials provided during runtime.